src/test/java/cc/robotdreams/API/ProjectManagementAPITest.java

```diff
@@ -0,0 +1,148 @@
+package cc.robotdreams.API;
+
+import automation.app.api.KanboardAPI;
+import automation.base.BaseAPITest;
+import io.restassured.response.Response;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.hamcrest.Matchers;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.*;
+
+@Test(priority = 0)
+public class ProjectManagementAPITest extends BaseAPITest {
+    @Test(groups = {"createProject", "smoke"}, priority = 0)
+    public void createProject() {
+        //Setup
+        String projectName = "Project_" + RandomStringUtils.random(3, false, true);
+        Integer ownerID = 1;
+
+        logger.info("Random project name to create: " + projectName);
+
+        //Test
+        Response createProjectResult = KanboardAPI.createProject(projectName, requestID, ownerID, true);
```

Comment:
The `requestID` here always is `1`. It should be hidden inside method `KanboardAPI.baseRequestParams` because you do not need to test jsonrps protocol wrapper.